astroquery/query.py

```diff
@@ -10,20 +10,34 @@
 import io
 import os
 import requests
+import email.utils as eut
```

Comment:
maybe keep this with the full name, as it's not a frequently used thing to any of us reasonably remember what eut stands for.
```suggestion
import email.utils
```